package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
     "docs": "pulp docs",
-    "build": "pulp build -- --censor-lib --strict",
```

Comment:
Thanks for having a look at it @joneshf 

I assumed that the idea was to be as strict as possible and that in the past there were some warnings in the libs. Since there are no warnings in the libs now I removed them.

But happy to change it back if that was the wrong assumption.